intg/src/main/java/org/apache/atlas/model/tasks/AtlasTask.java

```diff
@@ -239,6 +244,26 @@ public String getEntityGuid() {
         return entityGuid;
     }
 
+    public void setAssetsCountToPropagate(Long assetsCount) {
+        this.assetsCountToPropagate = assetsCount;
+    }
+
+    public Long getAssetsCountToPropagate() {
+        return assetsCountToPropagate;
+    }
+
+    public void setAssetsCountPropagated(Long assetsCountPropagated) {
+        this.assetsCountPropagated = assetsCountPropagated;
+    }
+
+    public Long getAssetsCountPropagated(){
+        return  assetsCountPropagated;
+    }
+
+    public void incrementAssetCountPropagated() {
```

Comment:
Let's remove this method. We are incrementing by `CHUNK_SIZE` by using the `setAssetsCountPropagated` method instead.

Reply:
removed, not needed and not used anywhere in code